receiver/oracledbreceiver/documentation.md

```diff
@@ -0,0 +1,51 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# oracledbreceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| **oracledb.session.cpu_usage** | CPU usage (%) | 1 | Gauge(Double) | <ul> </ul> |
+| **oracledb.session.enqueue_deadlocks** | Total number of deadlocks between table or row locks in different sessions. | 1 | Gauge(Int) | <ul> </ul> |
```

Comment:
I believe we usually use `{deadlocks}` as a unit for such counters. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md as example

Reply:
done, added for all counts.